app/services/idv/vendor_result.rb

```diff
@@ -0,0 +1,26 @@
+module Idv
+  class VendorResult
+    attr_reader :success, :errors, :reasons, :session_id, :normalized_applicant
+
+    def self.new_from_json(json)
+      parsed = JSON.parse(json, symbolize_names: true)
+
+      applicant = parsed[:normalized_applicant]
+      parsed[:normalized_applicant] = Proofer::Applicant.new(applicant) if applicant
+
+      new(**parsed)
+    end
+
+    def initialize(success:, errors:, reasons:, session_id:, normalized_applicant:)
+      @success = success
+      @errors = errors
+      @reasons = reasons
+      @session_id = session_id
+      @normalized_applicant = normalized_applicant
+    end
+
+    def success?
+      success
+    end
+  end
+end
```

Comment:
I agree with Reek's TooManyInstanceVariables offense. How about making this class only accept the `result` as an argument, and then the class can define public method for success, errors, reasons, session_id, and normalized_applicant?